### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by RobSteuer** [bonomani/devmon#87] [General]
Hi, I see progress and updates are available for devmon, many thanks and great to see!

I do have a question as I'd like to deploy the latest version.  The INSTALLATION in docs seems to be from many years ago, still references the URL: http://www.switch.ch/misc/leinen/snmp/perl/ which is not available.  Is there any updated single-node server documentation on pre-requisites and installation?

**Comment by RolandRosenfeld** [comment]
I'm not sure about an updated documentation, but the missing SNMP Session module can now be found at https://github.com/sleinen/snmp-session

**Comment by bonomani** [comment]
Hi Rob, 
Thanks for reporting, will try to update the doc (The doc was not updated for years, but I will if this is needed)
In most Linux distribution you should install with your favorite package manager (the simplest solution): perl-SNMP_Session
There is also a Perl module on CPAN (second solution): SNMP_Session 
And finally you can also your download it (Roland indicated the current source of the module) which was relocated to GitHub some years ago. (seems to me)
Please let me know if this resolve you problem or not (also could be kind if you can tell me which method you use and which OS you have) so I can close the discussion

**Comment by RobSteuer** [comment]
Hi bonomani - 

Sorry to come back to this so late, I had to put this on the back burner for a while.  I got everything working, was switching from an older CentOS to Ubuntu 22.04.  So there were some differences between the linux versions, but after I installed libsnmp-perl and libsnmp-session-perl in Ubuntu, all was good.  

Thank you!

**Reply by bonomani** [reply]
Cool , and thanks to you for reporting anything that make it better!
